docs/source/gbd2019_models/causes/diarrheal_diseases/index.rst

```diff
@@ -0,0 +1,103 @@
+.. _2019_cause_diarrhea:
+
+==================
+Diarrheal Diseases
+==================
+
+Disease Description
+-------------------
+
+
+
+
+Modeling Diarrheal Diseases in GBD 2019
+---------------------------------------
+
+The GBD diarrheal diseases model includes a cause of death (CoD) model and a 
+nonfatal model, each of which is split into diarrhea etiologies.
+
+The CoD model used data from vital registration systems, surveillance systems, 
+and verbal autopsy. Deaths were modelled separately for males and females and 
+for under 5 year olds vs. the rest of the population using CODEm models with 
+country-level covariates.
+
+The nonfatal model used diarrhea incidence and prevalence data in community and 
+hospital settings in addition to hospital and claims data tagged ICD9 codes 
+001-009.9 and ICD10 codes A00-A09. All hospital and claims data was converted 
+to prevalence (or point prevalence) data using a mean duration of 4.3 (4.2-4.4) 
+days. Data was then adjusted for seasonality, and all non-reference case data 
```

Comment:
ooh, should we include seasonality in our CIFF model?  probably not at this stage, but cool to know that GBD has some evidence we could use if/when we wanted to incorporate this.

Reply:
there is some seasonality for wasting. It could be a complexity to add later once we finish the simpler model.